inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/topic/TopicTdmController.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.tubemq.manager.controller.topic;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.inlong.tubemq.manager.controller.TubeMQResult;
+import org.apache.inlong.tubemq.manager.entry.TopicEntry;
+import org.apache.inlong.tubemq.manager.entry.TopicStatus;
+import org.apache.inlong.tubemq.manager.exceptions.TubeMQManagerException;
+import org.apache.inlong.tubemq.manager.repository.TopicRepository;
+import org.apache.inlong.tubemq.manager.service.TopicBackendWorker;
+import org.apache.inlong.tubemq.manager.service.TopicFuture;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+
+@RestController
+@RequestMapping(path = "/business")
+@Slf4j
+public class TopicTdmController {
```

Comment:
Excuse, what does the `Tdm` mean?

Reply:
@healchow do we still have the `business` concept?

Reply:
> Excuse, what does the `Tdm` mean?

tdm is the inlong manager

Reply:
> @healchow do we still have the `business` concept?

We now use `inlong group` as data grouping and no longer support `business` concept.

Reply:
> tdm is the inlong manager

@bluewang please change another name, tdm is so weird.